manifests/s/Saladict/app/3.0.6/Saladict.app.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Saladict.app
```

```suggestion
PackageIdentifier: allentown521.Saladict
```

Package identifiers should adhere to <publisher>.<package name> format.
软件包标识符应遵守 <发布者>.<软件包名称> 的格式